### PR TITLE
Fix when sometimes captured widget wasn't released.

### PR DESCRIPTION
**Pull request by rsredsq**
There's also a fix for #491, added "Material" label

### CODE REVIEW

File: Script/AtomicEditor/ui/frames/inspector/AttributeInfoEdit.ts [hunk 1]
```diff
@@ -718,6 +718,7 @@ class ResourceRefAttributeEdit extends AttributeInfoEdit {
 
         var lp = new Atomic.UILayoutParams();
         lp.width = 140;
+        lp.height = 24;
```
Comment (JoshEngebretson) [inline Script/AtomicEditor/ui/frames/inspector/AttributeInfoEdit.ts]: Thanks :)

File: Script/AtomicEditor/ui/frames/inspector/ComponentAttributeUI.ts [hunk 2]
```diff
@@ -131,10 +131,11 @@ class SubmeshAttributeEdit extends AttributeInfoEdit {
 
     createMaterialEdit(materialIndex: number) {
 
-        var o = InspectorUtils.createAttrEditFieldWithSelectButton("", this.mainLayout);
+        var o = InspectorUtils.createAttrEditFieldWithSelectButton("Material", this.mainLayout);
```
Comment (JoshEngebretson) [inline Script/AtomicEditor/ui/frames/inspector/ComponentAttributeUI.ts]: Whoa, easy peasy!

File: Script/AtomicEditor/ui/frames/inspector/ComponentAttributeUI.ts [hunk 3]
```diff
@@ -429,10 +430,10 @@ class SubmeshListAttributeEdit extends AttributeInfoEdit {
                     submeshEdit.initialize(this.editType, this.attrInfo);
                     this.layout.addChild(submeshEdit);
                     this.submeshEdits[name] = submeshEdit;
+                    InspectorUtils.createSeparator(this.layout);
```
Comment (JoshEngebretson) [inline Script/AtomicEditor/ui/frames/inspector/ComponentAttributeUI.ts]: :+1:

File: Source/ThirdParty/TurboBadger/tb_widgets.cpp [hunk 4]
```diff
@@ -1353,19 +1353,29 @@ void TBWidget::InvokePointerDown(int x, int y, int click_count, MODIFIER_KEYS mo
...
+        if (captured_widget)
+        {
+            captured_widget->ConvertFromRoot(x, y);
+            if (!cancel_click && captured_widget->GetHitStatus(x, y))
+            {
+                captured_widget->InvokeEvent(ev_click);
+            }
+            captured_widget->InvokeEvent(ev_up);
+            captured_widget->ReleaseCapture();
+        }
```
Comment (JoshEngebretson) [inline Source/ThirdParty/TurboBadger/tb_widgets.cpp]: This is a little hard to parse as the formatting messed it up, what was the bug here?

**Comment by JoshEngebretson** [comment]
Looks great, wondering about the bug that was fixed, will pass to @LaraEngebretson to do some testing on it

File: Source/ThirdParty/TurboBadger/tb_widgets.cpp (hunk 4 above)
Comment (rsredsq) [inline Source/ThirdParty/TurboBadger/tb_widgets.cpp]: There was a check for captured_widget only if down_widget exists, so need to check if they should be uncaptured separately

**Comment by JoshEngebretson** [comment]
Tests out, LANDING!  Thanks :+1:
